pkg/auth/auth.go

```diff
@@ -97,6 +97,51 @@ func (c *Client) GetSession(ctx context.Context) (*session.Token, error) {
 	return tok, nil
 }
 
+// GetSessions returns all session tokens. It attempts to refresh any
+// expired tokens, and returns an error if any tokens are unable to be
+// refreshed. If there are errors, it still returns all tokens, some of which
+// may be expired.
+func (c *Client) GetSessions(ctx context.Context) ([]*session.Token, error) {
```

Comment:
where is this used?

Reply:
not currently used, but the plan is for envsec to use it to show what orgs you can use to initialize a project.